### PR TITLE
Fix #50

### LINKED ISSUE #50: Remove unused UI code
A bunch of the prompt panels in the codebase are not used. Removing them makes the tests run faster.

### PR DESCRIPTION
**Pull request by diracdeltas**
cc @adon-at-work for review

### CODE REVIEW

**Comment by adon-at-work** [comment]
will do this week

File: src/javascript/crypto/e2e/extension/ui/prompt/prompt.js [hunk 1]
```diff
@@ -365,27 +216,20 @@ ui.Prompt.prototype.renderMenu_ = function(elem, blob, opt_showReduced) {
...
           this.pgpLauncher_.start(passphrase);
-          if (contentBlob && contentBlob.action) {
-            if (contentBlob.action == constants.Actions.GET_PASSPHRASE) {
-              contentBlob.action = undefined; // Fall into the default action.
-            }
-            this.processSelectedContent_(contentBlob, contentBlob.action);
-          } else {
-            this.close();
-          }
+          this.processSelectedContent_();
```
Comment (adon-at-work) [inline src/javascript/crypto/e2e/extension/ui/prompt/prompt.js:228]: should be `this.close()` unless it's intended to change the original behavior (i.e., closing the dialog after correct passphrase is entered).

**Comment by adon-at-work** [comment]
i skipped reviewing most of the deleted files (over 2000 lines), and believe that it won't affect the user workflow. other things look good to me. :+1: 

p.s. you may like also cleaning up those `promptShareTitle`, `promptWhatever` messages that you removed in 
`src/javascript/crypto/e2e/extension/_locales/en/messages.json`

File: src/javascript/crypto/e2e/extension/ui/prompt/prompt.js (hunk 1 above)
Comment (diracdeltas) [inline src/javascript/crypto/e2e/extension/ui/prompt/prompt.js:228]: Good catch! Added this back.

**Comment by diracdeltas** [comment]
Awesome, addressed review comments. Feel free to merge or let me know if you don't have merge access.

**Comment by adon-at-work** [comment]
good

**Pull request merged**